src/e2e/step-definitions/zaak.ts

```diff
@@ -201,9 +201,9 @@ Then(
   async function (this: CustomWorld, user1: string, _user2: string) {
```

Comment:
user1 and user2, those are horrible names for parameters, don't we know the context of these users?

Reply:
Don't shoot the messenger @bas-info-nl  ;-) hahaha

I don't know and not intending to find out ATM; Rick's party it was :-/ - the thing is that I only changed the Dutch naming to English; I prefer to leave it for now;

Reply:
it was just something I noticed, I know it's "old" code...  up to you, what to do with this..

Reply:
✅  I needed to revert the renaming anyway; so better not to touch this file at all anymore